ext/util/math.h

```diff
@@ -19,6 +19,7 @@
 
 void import_twos_complement(cass_byte_t *data, size_t size, mpz_t *number);
 cass_byte_t *export_twos_complement(mpz_t number, size_t *size);
+int prepare_string_conversion(char *in, int *pos, int *negative);
```

Comment:
This doesn't seem to be used anywhere other than `math.c`. It doesn't need to be expose and could be mark `static`.